chatstripe/payment/views.py

```diff
@@ -1,3 +1,15 @@
 from django.shortcuts import render
+from rest_framework import serializers
+from rest_framework.generics import ListCreateAPIView
+from .models import CardDetail
+from .serializers import CardDetailSerializer
+
+
+class CardDetailView(ListCreateAPIView):
```

Comment:
generic ListCreateAPIView is OK but in most of the cases we use APIView, it would be better to follow those standards.
Please change it into APIVIEW with a get, post, and put methods defined within it.